diff --git a/db/seeds/shared/admin.seed.rb b/db/seeds/shared/admin.seed.rb
index 6cace892..67ba354f 100644
--- a/db/seeds/shared/admin.seed.rb
+++ b/db/seeds/shared/admin.seed.rb
@@ -4,7 +4,8 @@ attributes = {
   last_name: 'Admin',
   password: 'PleaseChangeYourPassword',
   name_abbreviation: 'ADM',
-  type: 'Admin'
+  type: 'Admin',
+  account_active: true
 }
 
 User.create!(attributes) unless User.find_by(type: 'Admin', name_abbreviation: 'ADM')
