diff --git a/app/views/layouts/application.haml b/app/views/layouts/application.haml
index 81575617..29173ef3 100644
--- a/app/views/layouts/application.haml
+++ b/app/views/layouts/application.haml
@@ -4,6 +4,7 @@
     %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
     %title #{ENV['APPLICATION_TITLE'].presence || 'Chemotion'}
     = stylesheet_link_tag    'application', media: 'all'
+    %link{:rel => :stylesheet, :type => :"text/css", :href => "/assets/custom.css"}
     = javascript_pack_tag 'application'
     = csrf_meta_tags
   %body
